File: TransCoder_tokenized_test_set_functions/java/PROGRAM_FIND_LINE_PASSING_2_POINTS.java
static void lineFromPoints ( Pair P, Pair Q ) {
  int a = Q . second - P . second;
  int b = P . first - Q . first;
  int c = a * ( P . first ) + b * ( P . second );
  if ( b < 0 ) {
    System . out . println ( "The line passing through points P and Q is: " + a + "x " + b + "y = " + c );
  }
  else {
    System . out . println ( "The line passing through points P and Q is: " + a + "x + " + b + "y = " + c );
  }
}

File: TransCoder_tokenized_test_set_functions/cpp/PROGRAM_FIND_LINE_PASSING_2_POINTS.cpp
pair < double, double > void lineFromPoints ( pdd P, pdd Q ) {
  double a = Q . second - P . second;
  double b = P . first - Q . first;
  double c = a * ( P . first ) + b * ( P . second );
  if ( b < 0 ) {
    cout << "The line passing through points P and Q is: " << a << "x " << b << "y = " << c << endl;
  }
  else {
    cout << "The line passing through points P and Q is: " << a << "x + " << b << "y = " << c << endl;
  }
}